src/components/SplashScreen/index.tsx

```diff
@@ -0,0 +1,91 @@
9 17.1818L103.724 17.1895L103.731 17.8144V20.8544H102.581V15.0676H103.685L105.733 18.0536L106.079 18.5706L106.094 18.5629L106.079 17.9456V15.0676H107.222V20.8544ZM112.659 19.1184C112.659 20.2603 111.869 20.9625 110.527 20.9625C109.859 20.9625 109.246 20.8004 108.747 20.5304L108.916 19.4116C109.422 19.6816 109.944 19.836 110.511 19.836C111.14 19.836 111.416 19.6045 111.416 19.2033C111.416 18.2928 108.77 18.6786 108.77 16.742C108.77 15.7621 109.399 14.9673 110.856 14.9673C111.416 14.9673 112.022 15.0831 112.467 15.2683L112.306 16.3639C111.831 16.2096 111.355 16.117 110.918 16.117C110.204 16.117 110.013 16.3639 110.013 16.6725C110.013 17.5753 112.659 17.1586 112.659 19.1184ZM117.774 19.1184C117.774 20.2603 116.984 20.9625 115.642 20.9625C114.975 20.9625 114.361 20.8004 113.862 20.5304L114.031 19.4116C114.537 19.6816 115.059 19.836 115.627 19.836C116.256 19.836 116.532 19.6045 116.532 19.2033C116.532 18.2928 113.885 18.6786 113.885 16.742C113.885 15.7621 114.514 14.9673 115.972 14.9673C116.532 14.9673 117.138 15.0831 117.583 15.2683L117.422 16.3639C116.946 16.2096 116.47 16.117 116.033 16.117C115.32 16.117 115.128 16.3639 115.128 16.6725C115.128 17.5753 117.774 17.1586 117.774 19.1184ZM123.227 16.2404H121.678V20.8544H120.374V16.2404H118.832L118.863 15.0676H123.197L123.227 16.2404ZM125.842 19.7202H128.488L128.458 20.8544H124.546V15.0676H128.343L128.366 16.2173H125.842V17.3669H127.967V18.4008H125.842V19.7202ZM131.279 19.6816H133.65L133.619 20.8544H129.975V15.0676H131.279V19.6816ZM136.318 19.6816H138.688L138.658 20.8544H135.014V15.0676H136.318V19.6816ZM141.349 19.7202H143.995L143.965 20.8544H140.053V15.0676H143.85L143.873 16.2173H141.349V17.3669H143.474V18.4008H141.349V19.7202ZM51.0976 29.9504C51.0976 30.9766 50.6221 31.5707 49.3641 31.5707H47.2547V25.7839H49.2491C50.3153 25.7839 50.8522 26.2391 50.8522 27.2036C50.8522 27.9597 50.4917 28.361 50.0084 28.4998V28.5153C50.6834 28.6542 51.0976 29.094 51.0976 29.9504ZM50.5991 29.9118C50.5991 29.1248 50.1849 28.7622 49.3411 28.7622H47.738V31.1232H49.3718C50.1388 31.1232 50.5991 30.8068 50.5991 29.9118ZM50.369 27.2885C50.369 26.5401 50.0084 26.2391 49.2337 26.2391H47.7457V28.3455H49.2951C50.0161 28.3455 50.369 27.9906 50.369 27.2885ZM53.6817 31.0923H56.7346L56.7116 31.5707H53.1832V25.7839H56.5735L56.5965 26.2623H53.6817V28.3224H56.2053V28.7545H53.6817V31.0923ZM62.2181 27.3502C62.2181 28.2915 61.7886 28.7853 60.9908 28.8779L62.594 31.5707H61.9803L60.5076 28.9705H59.1729V31.5707H58.6667V25.7839H60.5306C61.6888 25.7839 62.2181 26.2931 62.2181 27.3502ZM61.6965 27.3734C61.6965 26.5632 61.336 26.2391 60.5306 26.2391H59.1729V28.5539H60.4846C61.3207 28.5539 61.6965 28.2144 61.6965 27.3734ZM64.8559 31.0769H67.7093L67.6863 31.5707H64.3496V25.7839H64.8559V31.0769ZM69.557 31.5707V25.7839H70.0633V31.5707H69.557ZM76.7127 31.5707H76.3368L73.3607 27.0956L73.0232 26.5709L73.0079 26.5786L73.0155 27.2036V31.5707H72.5553V25.7839H73.0539L75.9303 30.151L76.2678 30.6757L76.2832 30.668L76.2678 30.0507L76.2601 25.7839H76.7127V31.5707Z"
```

Comment:
For now, this is okay. In my branch, I added an Icons folder where we will place those SVGs. Right now it kind of pollutes the whole file. But as I said, fine for now! 👍